wrapper.ts

```diff
@@ -209,7 +210,7 @@ function setupMethods (soljson) {
   }
 
   // Expects a Standard JSON I/O but supports old compilers
-  const compileStandardWrapper = function (input, readCallback) {
+  const compileStandardWrapper = function (input, readCallback?: any) {
```

Comment:
I'm fine merging it assuming these types currently are only relevant intra-package and do not affect users of the library.

Reply:
i'm 99.9% sure are that these types are not exported for the consumer yet, e.g you can test with `npm pack` and see if any type files are outputted. I will just validate tomorrow.

Types are effectively all internal until we flag that they can be exported.

Reply:
At the moment *everything* is `any` type because solc is JS, not TS, and doesn't ship any definitions.  A common strategy for migrating to TS is to make everything `any` just to get the build pipeline setup and then you can iteratively improve on the types as you go and it becomes easy for people to submit PRs

Reply:
^ this point is true but I was regarding about the type definition files which would be generated, since we have any and ? specified these type files can be generated and would not be empty.

if 

```
 // Generate d.ts files
    "declaration": true,
    ````

was specified in the tsconfig then we could get files .d.ts files which can be included in the packaging process. Right now these don't exist (until we want them to be included).